string zone
string manual_operation_result
---
bool success